setup.py

```diff
@@ -45,11 +45,6 @@
     zip_safe=False,
     packages=find_packages(),
     include_package_data=True,
-    package_data={
-        '': [
-            'linchpin.constants',
-        ],
-    },
```

Comment:
so does this still get included?  I can't quite tell from the above changes what will get picked up and won't.